virttest/shared/scripts/pmsocat/pmsocat36.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+import argparse
+import asyncio
+import os
+import sys
+
+_args = None
+
+
+@asyncio.coroutine
+def forward(reader, writer, prefix):
+    total = 0
+    while True:
+        data = yield from reader.read(_args.blocksize)
+        if not data:
+            break
+        new_tot = total + len(data)
+        print(f'{ prefix }  length={ len(data) } from={ total } to={ new_tot - 1}')
+        total = new_tot
+        if _args.output:
+            print(data.decode(encoding='ascii', errors='replace'))
+        writer.write(data)
+        yield from writer.drain()
+    print(f'{ prefix } close')
```

Comment:
maybe use logging instead

Reply:
I think it's fine to call print here...it's an independent script to start unix/tcp service from local and remote, so I'd like to use print here... is there any specific reason to use logging?

Reply:
@Yingshun Do prints show up in the avocado test log?

Reply:
I think in general it's a good practice to use logging instead of print but if this an independent script, it seems fine.

It's for example used here: https://github.com/avocado-framework/avocado-vt/blob/f280747ffcea26bf1a90a7d5acbf7b0b94f679d2/virttest/utils_conn.py#L2073